myapp/views.py

```diff
@@ -77,10 +78,7 @@ def homework4(request):
     content['task5'] = 'done'
 
     # 6. Получить первые 2 комментария по дате создания к статье у которой имя автора последнее по алфавиту.
-    author = Profile.objects.order_by('name').last()
-    article = Article.objects.filter(author=author).last() #взял самую "свежую" статью автора
-    task6 = article.comments.all().order_by('date')[:2]
-
-    content['task6'] = task6
+    content['task6'] = Comment.objects.filter(content_type_id=15).order_by('date', '-article__author')[:2]
```

Comment:
Hardcoding values is a bad way in 99% of situations. 15 is a hardcoding

Reply:
I haven't noticed, sorry...
more than 1 hour of googling and the issue is fixed)